app/kebab-case-remover/kebabRemover.js

```diff
@@ -0,0 +1,17 @@
+class KebabRemover {
+    execute(queryResult) {
+        const queryKeys = Object.keys(queryResult);
+        const camelize = s => s.replace(/_./g, x=>x[1].toUpperCase());
+        const queryKeysCamel =  queryKeys.map(camelize);
+        const updatedObject = {};
+        
+        Object.keys(queryResult).forEach((key, index) => {
+          const newKey = queryKeysCamel[index];
+          updatedObject[newKey] = queryResult[key];
+        });
+
+        return updatedObject;
+    }
```

Comment:
Looking forward for the moment u explain me every line of this :)

Reply:
it's easy to be honest. I googled something like 'js kebab to camel' and opened few first links. Then combined some of them and voila 🤓